package.json

```diff
@@ -9,12 +9,13 @@
   },
   "scripts": {
     "start": "node src/app.js",
+    "go": "yarn build && yarn start",
```

Comment:
Is there a reason we need a new script for this rather than just adding build to the start command? I just find it really confusing to know which to use as we have so many

Reply:
Of course will take out now, that was just for speed during dev